pkgs/servers/dynomite/default.nix

```diff
@@ -0,0 +1,27 @@
+{ fetchFromGitHub, stdenv, autoreconfHook, libressl, git }:
+stdenv.mkDerivation rec {
+  name = "dynomite-${version}";
+  version = "0.6.14";
+
+  src = fetchFromGitHub {
+    owner = "Netflix";
+    repo = "dynomite";
+    rev = "67166e3cdc194caa16843c613d2bd9a0e86e86d3";
+    sha256 = "1lk6hrkad4zpfjvmn5hskzfr748zp7ccwqbhy46skw5hvsy2fa68";
+  };
+
+  buildInputs = [ autoreconfHook git libressl ];
+
+  NIX_CFLAGS_COMPILE = [
+    "-Wformat"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "
+    A generic dynamo implementation for different k-v storage engines
```

Comment:
I would just move the description into a single line.
`description = "A generic dynamo implementation for different k-v storage engines";`